frontend/projects/ui/src/app/modals/generic-input/generic-input.component.ts

```diff
@@ -52,6 +77,7 @@ export class GenericInputComponent {
     try {
       await this.options.submitFn(value)
       this.modalCtrl.dismiss(undefined, 'success')
+      this.clearInput()
```

Comment:
Is this necessary? Are these values persisting after modal destroy?

Reply:
Never tested without it - would have to confirm.